create-chained-sessions.sh

```diff
@@ -15,15 +15,16 @@ declare REGION='us-east-1'
 # Args:
 # 1: name of environment ("playground", "staging", etc.)
 # 2: sso role name to use for the parent session
-# 3: scope of the IAM role ("panorama" or "eks").
+# 3: scope of the IAM role ("panorama" or "eks" or "rds").
 # 4: name of the persona (e.g. admin, dev-writer, etc.) the new session is for
 function createLeappSession {
-    green_echo "creating chained session for $1 with persona $4"
+    green_echo "creating chained session for $1 with persona $5"
     environment_name=$1
-    parent_session_name="panorama-k8s-${environment_name}"
-    parent_role_name=$2
-    iam_role_scope=$3
-    persona_name=$4
+    service_name=$2
```

Comment:
This is yet another ordered arg, but I figured we don't care about adding another arg since this function is called within the script at the bottom.